contracts/base/Consensus.sol

```diff
@@ -60,6 +60,27 @@ contract Consensus is Base, OwnerSignersRole {
         _;
     }
 
+    /**
+        @notice Checks if the number of responses is greater or equal to a percentage of
+        the number of signers.
+     */
+    modifier onlyEnoughSubmissions(uint256 responseCount) {
+        SettingsInterface settings = _getSettings();
+        uint256 percentageRequired = settings
+            .platformSettings(settings.consts().REQUIRED_SUBMISSIONS_PERCENTAGE_SETTING())
+            .value;
+        // https://stackoverflow.com/questions/17005364/dividing-two-integers-and-rounding-up-the-result-without-using-floating-point
+        uint256 minimumResponseCount = _signerCount.mul(percentageRequired).add(9999).div(
+            10000
+        );
```

Comment:
I don't think this is correct:
```
signers = 6
percentRequired = 8000

signers * percentRequired + 9999 / 10000 = 48000.9999
```

We could use `NumbersLib.ratioOf()` to get the ratio of signers from the `responseCount` and then compare it to the platform setting. That way we don't have to worry about the rounding issue of getting 80% of 6 which would chop off the decimals.

Heres what I'm thinking:
```
uint256 signersRatio = responseCount.ratioOf(_signersCount);
// signersRatio = 6 * 10000 / 8 = 7500
require(signersRatio >= percentageRequired);
```